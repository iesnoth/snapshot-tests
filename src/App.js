import logo from './logo.svg';
import './App.css';

function App() {
  return (
    <div>
      learn react
      <h2>I updated the snapshot</h2>
    </div>
  );
}

export default App;
