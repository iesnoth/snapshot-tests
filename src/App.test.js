import { render, screen } from '@testing-library/react';
import App from './App';
import renderer from 'react-test-renderer'

test('renders a snapshot', () =>{
  //this variable is usually called "tree"
  //saving it to JSON makes it easier to save and compare to other snapshots
  const tree = renderer.create(<App />).toJSON()
  expect(tree).toMatchSnapshot()
})
