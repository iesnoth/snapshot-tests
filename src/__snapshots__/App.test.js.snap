// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div>
  learn react
  <h2>
    I updated the snapshot
  </h2>
</div>
`;
